doc_classes/OpenXRFbBodyTrackingExtensionWrapper.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="OpenXRFbBodyTrackingExtensionWrapper" inherits="OpenXRExtensionWrapperExtension" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/godotengine/godot/master/doc/class.xsd">
 	<brief_description>
+		Wraps the [code]XR_FB_body_tracking[/code] extension.
```

Comment:
Should we unregister the singleton for the extension wrapper classes that don't provide any methods?

Reply:
The singleton isn't being registered, only the class, but that's enough for it to appear in the editor's help docs so we need to provide something here. And, unfortunately, we have to register the class, because unlike in Godot itself, GDExtension classes won't work without being registered.